defrag/modules/utils.py

```diff
@@ -0,0 +1,91 @@
+# Defrag - centralized API for the openSUSE Infrastructure
```

Comment:
I think it's a bad idea to have these utils as a module. Could you move them to a `helpers` or `utils` folder similar to #6 ?

Reply:
did that

Reply:
It is ok for you now or you want me to change something?